test/uk/gov/hmrc/tai/model/rti/RtiPaymentSpec.scala

```diff
@@ -16,16 +16,16 @@
 
 package uk.gov.hmrc.tai.model.rti
 
-import org.joda.time.LocalDate
+import java.time.LocalDate
 import org.scalatestplus.play.PlaySpec
 
 class RtiPaymentSpec extends PlaySpec {
 
   "RtiPayment compare method" must {
     "implement comparison behaviour" in {
-      rtiPayment1.compare(rtiPayment2) mustBe -1
+      rtiPayment1.compare(rtiPayment2) mustBe -2
       rtiPayment1.compare(rtiPayment2.copy(paidOn = LocalDate.now().minusDays(7))) mustBe 0
-      rtiPayment1.compare(rtiPayment2.copy(paidOn = LocalDate.now().minusDays(9))) mustBe 1
+      rtiPayment1.compare(rtiPayment2.copy(paidOn = LocalDate.now().minusDays(9))) mustBe 2
```

Comment:
because the previous compare would only return -1, 0, 1. CompareTo also check the difference

Reply:
is this functionality that was expected to change ?

Reply:
it's expected from the library's perspective, because this is used just for sorting, it shouldn't change the behaviour of any fucntions where it's used